stdlib/2and3/turtle.pyi

```diff
@@ -0,0 +1,487 @@
+    def onkey(self, fun: Callable[[], Any], key: str) -> None: ...
+    def listen(self, xdummy: Optional[float] = ..., ydummy: Optional[float] = ...) -> None: ...
+    def ontimer(self, fun: Callable[[], Any], t: int = ...) -> None: ...
+    @overload
+    def bgpic(self) -> str: ...
+    @overload
+    def bgpic(self, picname: str) -> None: ...
+    @overload
+    def screensize(self) -> Tuple[int, int]: ...
+    @overload
+    def screensize(self, canvwidth: int, canvheight: int, bg: Optional[_Color] = ...) -> None: ...
+    onscreenclick = onclick
+    resetscreen = reset
+    clearscreen = clear
+    addshape = register_shape
+    if sys.version_info >= (3,):
+        def onkeypress(self, fun: Callable[[], Any], key: Optional[str] = ...) -> None: ...
+        onkeyrelease = onkey
+
+class TNavigator(object):
+    START_ORIENTATION: Dict[str, Vec2D] = ...
+    DEFAULT_MODE: str = ...
+    DEFAULT_ANGLEOFFSET: int = ...
+    DEFAULT_ANGLEORIENT: int = ...
+    def __init__(self, mode: str = ...) -> None: ...
+    def reset(self) -> None: ...
+    def degrees(self, fullcircle: float = ...) -> None: ...
+    def radians(self) -> None: ...
+    def forward(self, distance: float) -> None: ...
+    def back(self, distance: float) -> None: ...
+    def right(self, angle: float) -> None: ...
+    def left(self, angle: float) -> None: ...
+    def pos(self) -> Vec2D: ...
+    def xcor(self) -> float: ...
+    def ycor(self) -> float: ...
+    @overload
+    def goto(self, x: Tuple[float, float]) -> None: ...
+    @overload
+    def goto(self, x: float, y: float) -> None: ...
+    def home(self) -> None: ...
+    def setx(self, x: float) -> None: ...
+    def sety(self, y: float) -> None: ...
+    @overload
+    def distance(self, x: Union[Turtle, Tuple[float, float]]) -> float: ...
+    @overload
+    def distance(self, x: float, y: float) -> float: ...
+    @overload
+    def towards(self, x: Union[Tuple[float, float]]) -> float: ...
+    @overload
+    def towards(self, x: float, y: float) -> float: ...
+    def heading(self) -> float: ...
+    def setheading(self, to_angle: float) -> None: ...
+    def circle(self, radius: float, extent: Optional[float] = ..., steps: Optional[int] = ...) -> None: ...
+    fd = forward
+    bk = back
+    backward = back
+    rt = right
+    lt = left
+    position = pos
+    setpos = goto
+    setposition = goto
+    seth = setheading
+
+
+class TPen(object):
+    def __init__(self, resizemode: str = ...) -> None: ...
+    def resizemode(self, rmode: Optional[str] = ...) -> None: ...
+    @overload
+    def pensize(self) -> int: ...
+    @overload
+    def pensize(self, width: int) -> None: ...
+    def penup(self) -> None: ...
+    def pendown(self) -> None: ...
+    def isdown(self) -> bool: ...
+    @overload
+    def speed(self) -> int: ...
+    @overload
+    def speed(self, speed: _Speed) -> None: ...
```

Comment:
I updated the `_Speed` type alias to use float instead of int. (Note: we use `_Speed` only in the argument type, so I think it's fine to broaden that alias.)